src/components/site/tableOfContents/index.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import reactn from 'reactn';
+import { Link, graphql, StaticQuery } from 'gatsby';
+import PropTypes from 'prop-types';
+import { Drawer } from 'react-md';
+import styles from './table-of-contents.module.scss';
+
+// import Footer from '../components/site/footer';
+const TO_PREFIX = '/';
+const routes = [
+  {
+    label: 'home',
+    to: `${TO_PREFIX}`,
+    icon: 'home',
+    exact: 'true',
+    primaryText: 'Home',
+    active: false,
+  },
+  {
+    label: 'style-guide',
+    to: `${TO_PREFIX}StyleGuide`,
+    icon: 'style',
+    primaryText: 'Style Guide',
+    active: false,
+  },
+  { divider: true },
+  {
+    primaryText: 'Table of Contents',
+    subheader: true,
+  },
+];
+
+@reactn
+class TableOfContents extends React.PureComponent {
+  // constructor(props) {
+  //   super(props);
+  //
+  // }
+
+  componentDidMount() {
+    const { navLinks } = this.props;
+    const links = routes.concat(navLinks);
```

Comment:
No change necessary but thought I would point out that this can also be written using the spread operator:
```
const links = [...this.routes, ...navlinks]
```
which is just kinda cool.

Reply:
woah! That's right! I completely forgot about this beautify little gem provided from those spread operators....really need to start utilizing those more.